tools/opensnoop.py

```diff
@@ -107,6 +107,10 @@
 #endif
 BPF_HASH(infotmp, u64, struct val_t);
 BPF_PERF_OUTPUT(events);
+"""
+
+bpf_text_kprobe = """
+BPF_HASH(infotmp, u64, struct val_t);
```

Comment:
This BPF_HASH is redundant. This will cause compilation error for kprobe case.

Reply:
ugh, I must have consused my versions and pushed the wrong one,
I recall fixing this, sry.. will fix in next version